src/controller/CHIPDeviceController.h

```diff
@@ -69,7 +70,7 @@ namespace Controller {
 
 using namespace chip::Protocols::UserDirectedCommissioning;
 
-constexpr uint16_t kNumMaxActiveDevices = 64;
+constexpr uint16_t kNumMaxActiveDevices = CHIP_CONFIG_CONTROLLER_MAX_ACTIVE_DEVICES;
```

Comment:
Follow-up: This should utilize the new pool constructs to be introduced [here](https://github.com/project-chip/connectedhomeip/pull/9748/files#), that will then in-turn, permit platforms to set this to 0 and automatically use the heap to allocate. Would you be able to help tackle that after that PR goes in?

Reply:
Yes, I can take a look at the pool for this use case when that PR goes in.